microsoft-edge/webview2/get-started/winui.md

```diff
@@ -386,6 +386,27 @@ As an example, next, you add scripts that send an alert when a user tries to ope
 Congratulations, you built your first WebView2 app!
 
 
+<!-- ====================================================================== -->
+## WinUI 3 WebView2 special considerations
+
+### SmartScreen
+
+WebView2 sends URLs that are navigated to in your application to the [SmartScreen](/windows/security/threat-protection/microsoft-defender-smartscreen/microsoft-defender-smartscreen-overview) service, to ensure that your customers stay secure. If you want to disable this navigation, you can do so via an environment variable:
+
+* `Environment.SetEnvironmentVariable("WEBVIEW2_ADDITIONAL_BROWSER_ARGUMENTS", "--disable-features=msSmartScreenProtection");`
+
+This environment variable must be set prior to `CoreWebView2` creation, which occurs when the [WebView2.Source property](https://docs.microsoft.com/windows/winui/api/microsoft.ui.xaml.controls.webview2.source) is set or the [WebView2.EnsureCoreWebView2Async method](https://docs.microsoft.com/windows/winui/api/microsoft.ui.xaml.controls.webview2.ensurecorewebview2async) is called.<!-- todo: replace links by correct platform's links -->
```

Comment:
@jasonstephen15 - what are the correct links, or a top-level link?  

Does the following page need to link to that top-level link?  https://docs.microsoft.com/en-us/microsoft-edge/webview2/webview2-api-reference#languages-and-frameworks
If the link is there, which link is it (which link text)?